tests/test_classifier/test_classification_report.py

```diff
@@ -189,6 +189,7 @@ def test_numpy_integration(self):
             "f1": {"unoccupied": 0.9800031994880819, "occupied": 0.9366447034972124},
         }
 
+    @pytest.mark.xfail(sys.platform == "win32", reason="images not close on windows")
```

Comment:
Usually we only add an `xfail` or `skip-if` decorator if the test has already run through CI and we have seen the specific failure case (e.g. it didn't work on a Windows virtual machine on Appveyor, or it didn't work on a Linus virtual machine using Miniconda on Travis). For now let's remove this decorator for now and let the test run as written.